### DISCUSSION TITLE
Change recaptcha language

### DISCUSSION THREAD

**Discussion by csgo-s** [pterodactyl/panel#3033] [Support] [answered]
How can I change the language of google recaptcha?

**Comment by DaneEveritt** [comment]
This is not possible.

**Reply by sinjs** [reply]
It is possible, I did it too.

**Comment by sinjs** [comment, marked as answer]
**Usually, reCAPTCHA automatically selects the language of the user's browser for the captcha. If you still want to change it, follow these steps:**

First of all, you need to install the panel modules for building. (Make sure you are in the `/var/www/pterodactyl` folder and have yarn installed!)
```shell
yarn install
```
Then you need to change the following file: `/var/www/pterodactyl/resources/scripts/components/auth/LoginContainer.tsx`.
In line 104 it should look like this:
```ts
sitekey={siteKey || '_invalid_key'}
onVerify={response => { 
```
between the sitekey and onVerify add this line:
```ts
hl={'[languagecode]'}
```
You will need to replace the `[languagecode]` with something like `en` (List [here](https://developers.google.com/recaptcha/docs/language))
After you've edited the file and saved it, go back to your Terminal and type this (Make sure you are in the `/var/www/pterodactyl` folder!):
```sh
yarn build:production
```
When the building is complete, the reCAPTCHA language should be changed to the language specified in `hl`.
